src/as-relation.c

```diff
@@ -1044,6 +1184,16 @@ as_relation_load_from_xml (AsRelation *relation, AsContext *ctx, xmlNode *node,
 		g_autofree gchar *side_str = (gchar*) xmlGetProp (node, (xmlChar*) "side");
 		priv->display_side_kind = as_display_side_kind_from_string (side_str);
 
+		g_free (priv->version);
+		priv->version = NULL;
+	} else if (priv->item_kind == AS_RELATION_ITEM_KIND_INTERNET) {
+		g_autofree gchar *bandwidth_str = (gchar*) xmlGetProp (node, (xmlChar*) "bandwidth_mbitps");
```

Comment:
`g_autofree gchar *bandwidth_str = as_xml_get_prop_value (node, "bandwidth_mbitps");`
(a bit nicer to read)